library/src/main/java/com/okta/oidc/clients/AuthClientImpl.java

```diff
@@ -59,14 +62,16 @@
     @AnyThread
     public void signIn(String sessionToken, AuthenticationPayload payload,
                        final RequestCallback<Result, AuthorizationException> cb) {
-        mDispatcher.execute(() -> {
+
+        mFutureTask = mDispatcher.submit(() -> {
```

Comment:
It this method can be run on any thread than `mFutureTask` should be `volatile`.

Reply:
sure will make it volatile